src/main/java/parsers/DateTimeParser.java

```diff
@@ -0,0 +1,31 @@
+package parsers;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class DateTimeParser {
+
+    /*
+     * Function to convert a String to LocalDateTime.
+     *
+     * @param dateTime a String to be read.
+     * @return The LocalDateTime object constructed from the String.
+     */
+    public static LocalDateTime parseDateTime(String dateTime) {
+        //2019-10-15 1800
+        DateTimeFormatter format = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
```

Comment:
This is not regarding code standard but I'd like to share what I did, i think i can be useful for you to consider. I have incorporated a list of all acceptable formats and checks whether the input aligns with any of the options. This makes the input format more flexible. Actually can ask GPT to list out for you the possible formats for datetime and date :)